### DISCUSSION TITLE
Is it possible to import extras for a URL .whl file?

### DISCUSSION THREAD

**Discussion by daniel-ji** [pyodide/pyodide#4790] [General]
Hi, 

Say I have an extra called `extra1` as part of my python package .whl. Is there any way to install the extras as well? Something like `await micropip.install("http://localhost:8000/my/package.whl[extra1]")`? Thank you!

**Comment by ryanking13** [comment]
We don't support that syntax now, but I think it would be good to support that syntax. Contributions are welcome.
